minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloCluster.java

```diff
@@ -167,4 +167,9 @@ public static ClientInfo getClientInfo(File directory) {
     Preconditions.checkArgument(clientProps.exists());
     return Accumulo.newClient().usingProperties(clientProps.getAbsolutePath()).info();
   }
+
+  @Override
+  public void close() throws Exception {
```

Comment:
I think it'd be better if this threw a more narrow exception, or didn't throw any checked exceptions at all (just RTEs).

Also, this is API, so we need to add inherit the parent javadoc and add a `@since 2.0.0` tag.